airflow/providers/microsoft/azure/hooks/data_factory.py

```diff
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
```

Comment:
Changing the path is OK but we can't drop the `hooks/azure_data_factory.py` yet because this is a breaking change.

The file should exist with deprecation notice and point to the updated hook.
See 
https://github.com/apache/airflow/blob/bff580602bc619afe1bee2f7a5c3ded5fc6e39dd/airflow/providers/amazon/aws/hooks/aws_dynamodb.py#L18-L28

Reply:
👍  Excellent point, thanks for the tip! I'll be more aware of changes like these in the future.